Update Auth endpoint (#1449)
onfido/onfido-sdk-ui: README.md

var Onfido = require('onfido-sdk-ui/dist/onfidoAuth.min.js')
```

> In addition to the alternative way of importing Auth, the integrator should also set the following code on the webpack/loader of their application:

Reviewer: ```suggestion
In addition to the alternative way of importing Auth, you should also set the following code in the loader (Webpack) of your application:
```

```javascript
new CopyPlugin({
  patterns: [
    {
      from: `../../node_modules/onfido-sdk-ui/dist/auth-sdk`,
      to: `${__dirname}/bin/src/auth-sdk`,
    },
  ],

> })

Reviewer 2: This would only work for Webpack. For any other implementation I think it's worth mentioning that they need to create a folder called `auth-sdk` in the public assets folder of their application and copy the files from `node_modules/onfido-sdk-ui/dist/auth-sdk` into it.

```

> This will fetch the core authentication technology from the SDK into your application. Using web workers for authentication enables the best performance achievable, without constraining the user usability.

Reviewer: ```suggestion
This will fetch the core authentication technology from the SDK into your application. Using web workers for authentication enables the best performance achievable, without constraining usability.
```
